### PR TITLE
Add hub icon and script to fix path of icon based on the $tfx_publisher

### PR DESCRIPTION
**Pull request by CloudyEMS**
* Added icon for the hub button
* Changed the `vss-extension.json` to show the icon
* Added nodejs script to update the path of the icon in `vss-extension.json` depending on the `$tfx_publisher`
* `package.json` now calls the script when the commands `package, package-debug, gallery-publish, gallery-publish-debug` are run.

### CODE REVIEW

File: Source/Extension.Voting/updateVssExtensionHubIcons.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+var vssExtensions = JSON.parse(fs.readFileSync('./vss-extension.json', 'utf-8'));
+
+publisherName = process.argv[2];
+if(!publisherName){
+    publisherName = vssExtensions.publisher;
+}
+
+const searchTerm = `.${vssExtensions.id}`;
+const replacement = `${publisherName}`;//.${vssExtensions.id}`;
```
Comment (manne) [inline Source/Extension.Voting/updateVssExtensionHubIcons.js]: is this comment needed? what's its purpose?

File: Source/Extension.Voting/updateVssExtensionHubIcons.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+for(let contribution of vssExtensions.contributions){
+    if(contribution.properties){
+        if(contribution.properties.iconAsset){
+            var str = contribution.properties.iconAsset;
+            contribution.properties.iconAsset = replaceString(str, searchTerm, replacement);//.replace(regex, replacement);
```
Comment (manne) [inline Source/Extension.Voting/updateVssExtensionHubIcons.js]: is this comment needed? what's its purpose?

File: Source/Extension.Voting/updateVssExtensionHubIcons.js (hunk 1 above)
Comment (oxxr) [reply Source/Extension.Voting/updateVssExtensionHubIcons.js]: Should be deleted

File: Source/Extension.Voting/updateVssExtensionHubIcons.js (hunk 2 above)
Comment (oxxr) [reply Source/Extension.Voting/updateVssExtensionHubIcons.js]: Should be deleted

**Comment by oxxr** [comment]
- Icon changed within another branch

**Pull request closed without merging**